README.rst

```diff
@@ -412,6 +412,7 @@ All join types are supported by |Brand|.
         .inner_join(join_table) \
         .outer_join(join_table) \
         .cross_join(join_table) \
+        .hash_join(join_table) \ # for google spanner, https://cloud.google.com/spanner/docs/query-execution-operators#hash-join
```

Comment:
I'd say we remove remove the comment altogether, I don't think we should link to a website.